elixir/apps/web/lib/web/live/policies/new.ex

```diff
@@ -1,15 +1,18 @@
 defmodule Web.Policies.New do
   use Web, :live_view
 
-  alias Domain.{Resources, Actors}
+  alias Domain.{Resources, Actors, Policies}
 
   def mount(_params, _session, socket) do
     with {:ok, resources} <- Resources.list_resources(socket.assigns.subject),
          {:ok, actor_groups} <- Actors.list_groups(socket.assigns.subject) do
+      form = to_form(Policies.Policy.Changeset.create_changeset(%{}, socket.assigns.subject))
```

Comment:
We should have a `new_policy(attrs \\ %{})` function in policies so that `web` app won't have to dig into changesets.

Reply:
Gotcha 👍

Reply:
Added the new function as recommended.